xsl/xccdf-report-impl.xsl

```diff
@@ -471,7 +471,10 @@ Authors:
         <xsl:when test="starts-with($href, 'http://iase.disa.mil/stigs/cci/')">
             <xsl:text>DISA CCI</xsl:text>
         </xsl:when>
-        <xsl:when test="starts-with($href, 'http://iase.disa.mil/stigs/srgs/')">
+        <xsl:when test="starts-with($href, 'http://iase.disa.mil/stigs/os/general/Pages/index.aspx')">
+            <xsl:text>DISA SRG</xsl:text>
+        </xsl:when>
+        <xsl:when test="starts-with($href, 'http://iase.disa.mil/stigs/app-security/app-servers/Pages/general.aspx')">
             <xsl:text>DISA SRG</xsl:text>
         </xsl:when>
         <xsl:when test="starts-with($href, 'http://iase.disa.mil/stigs/os/')">
```

Comment:
conflict here

Reply:
@mpreisler what is the conflict?

Reply:
there are 2 branches that will match the URL and both result in different shortnames. I know this is deterministic behavior and the first will match but I don't think you wanted to do that...

Reply:
suggestions? There is no longer a consistent URL for SRGs.

Reply:
Maybe nest it
```
<when .. starts-with 'http://iase.disa.mil/stigs/os/' ..
               <when test="startswith http://iase.disa.mil/stigs/os/general/Pages/index.aspx
                        DISA SRG
               </when>
              <otherwise>
                        DISA STIG
              </otherwise>
</when>
```

And don't remove `<xsl:when test="starts-with($href, 'http://iase.disa.mil/stigs/srgs/')">`